packages/eas-cli/src/commandUtils/easCommand.ts

```diff
@@ -0,0 +1,37 @@
+import { Command } from '@oclif/command';
```

Comment:
Rename the file `easCommand.ts` -> `EasCommand.ts`. It's a common pattern to have the file match the case of the exported class.